src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -55,29 +53,15 @@ const COLUMN_DATA: Column[] = [
 const AuditLogsTable = ({ showModel = false }: Props) => {
   const { modelName } = useParams<EntityDetailsRoute>();
   const dispatch = useAppDispatch();
-  const wsControllerURL = useSelector(getWSControllerURL);
   const auditLogs = useSelector(getAuditEvents);
   const auditLogsLoaded = useSelector(getAuditEventsLoaded);
   const auditLogsLoading = useSelector(getAuditEventsLoading);
-  const hasControllerConnection = useAppSelector((state) =>
-    getControllerConnection(state, wsControllerURL)
-  );
   const additionalEmptyMsg = showModel ? "" : ` for ${modelName}`;
   const emptyMsg = `There are no audit logs available yet${additionalEmptyMsg}!`;
   const columnData = COLUMN_DATA.filter(
     (column) => showModel || column.accessor !== "model"
   );
-
-  const [, setQueryParams] = useQueryParams<{ page: string | null }>({
-    page: null,
-  });
-
-  const fetchAuditEvents = useCallback(() => {
-    if (!wsControllerURL || !hasControllerConnection) {
-      return;
-    }
-    dispatch(jujuActions.fetchAuditEvents({ wsControllerURL }));
-  }, [dispatch, hasControllerConnection, wsControllerURL]);
+  const fetchAuditEvents = useFetchAuditEvents();
```

Comment:
I've moved all this to the `useFetchAuditEvents` so that it can also be called by the refresh button now that it's in the action bar.